build/config/webpack.base.js

```diff
@@ -31,7 +31,8 @@ export default {
     root,
     extensions: ['', '.js', '.jsx', '.json'],
     alias: {
-      'dtrace-provider': path.join(Const.BUILD_SYSTEM_DIR, 'config', 'empty_module.js'),
+      'dtrace-provider': 'empty-module',
+      'tofino-log-utils': 'empty-module',
```

Comment:
Bad path, should be "utils", not "config". Not sure how I missed this during the initial review.